helm-charts/infisical/templates/backend-deployment.yaml

```diff
@@ -70,3 +63,15 @@ spec:
     - protocol: TCP
       port: 4000
       targetPort: 4000 # container port 
+
+---
+
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ $backend.kubeSecretRef | default "infisical-backend" }}
+type: Opaque
+data:
+  {{- range $key, $value := $backend.secrets }}
```

Comment:
I'm curious, let's say I removed some secrets from the values file for backend. Would upgrading the helm chart also refect a one to one to this secret? Or do old values that no longer exist in the values file continue to exist but only only overwritten

Reply:
Awesome